### ISSUE TITLE
Did anyone able to apply Multi-class segmentation

### ISSUE THREAD

**Issue by gireeshkbogu** [qubvel/segmentation_models.pytorch#166]
The CamVid example is only for one class segmentation (car). Did anyone able to apply Multi-class segmentation? If not ay ideas other than changing the activation to softmax?

**Comment by pvthuy** [comment]
I tried to customize the code. What is your issues?

**Comment by geomags3** [comment]
Leave Sigmoid for segmentation in this example, because the multi-class mask is split into different layers.
E.g. (384, 480, N) if you've chosen N classes.

```python
ENCODER = 'se_resnext50_32x4d'
ENCODER_WEIGHTS = 'imagenet'
CLASSES = ['road', 'car', 'bicyclist']
ACTIVATION = 'sigmoid' # Leave Sigmoid for segmentation in this example
DEVICE = 'cuda'


# create segmentation model with pretrained encoder
model = smp.FPN(
    encoder_name=ENCODER, 
    encoder_weights=ENCODER_WEIGHTS, 
    classes=len(CLASSES), 
    activation=ACTIVATION
)

preprocessing_fn = smp.encoders.get_preprocessing_fn(ENCODER, ENCODER_WEIGHTS)
```
